locale/fr/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: index.hbs
 labels:
+  banner: Important security releases, please update now!
```

Comment:
If it saves a PR, I think we could go with:
"Mises à jour de sécurité importantes, veuillez mettre à jour!"
@nodejs/nodejs-fr Ca me semble redondant. D'autres idées?